beacon_api/api/info.py

```diff
@@ -26,10 +26,13 @@
         # TO DO implement some fallback mechanism for ID
         'id': '.'.join(reversed(host.split('.'))),
         'name': __title__,
+        "type": "urn:ga4gh:beacon",
         'description': __description__,
         'documentationUrl': __docs_url__,
+        "organization": __org_id__,
         'contactUrl': __org_contactUrl__,
-        'version': __version__
+        'version': __version__,
+        "extension": {}
```

Comment:
maybe here we can add information such as `"handover_support": true` - by default it would be `false` for CSC beacon, but I assume this can be left empty until we actually know what should be here.